src/debug/panel/_model_info_pane.template

```diff
@@ -1,5 +1,12 @@
+<%
+  var collapseAllClass = 'js-collapse-all-models TreeOptionsBar_item glyphicon glyphicon-';
+  collapseAllClass += w.allModelsCollapsed ? 'collapse-down' : 'collapse-up';
+%>
 <div class="js-model-pane pure-g">
   <div class="AppBody_panel pure-u-2-5">
+    <ul class="TreeOptionsBar">
```

Comment:
Can we switch this to a plain div since it doesn't actually have any list items?  Also in `_view_info_pane.template`.